Modules/System/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -51,20 +51,22 @@ codeunit 9999 "Upgrade Tag"
     /// <summary>
     /// Sets the upgrade tag to skipped.
     /// </summary>
-    /// <param name="ExistingTag">Tag code to set the skipped field</param>
-    procedure SetSkippedUpgrade(ExistingTag: Code[250])
+    /// <param name="ExistingTag">Tag code to set the Skipped Upgrade field</param>
+    /// <param name="SkipUpgrade">Sets the Skipped Upgrade field</param>
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; SkipUpgrade: Boolean): Boolean
     begin
-        UpgradeTagImpl.SetSkippedUpgrade(ExistingTag);
+        exit(UpgradeTagImpl.SetSkippedUpgrade(ExistingTag, SkipUpgrade));
     end;
 
     /// <summary>
     /// Sets the upgrade tag to skipped.
     /// </summary>
-    /// <param name="ExistingTag">Tag code to set the skipped field</param>
+    /// <param name="ExistingTag">Tag code to set the Skipped Upgrade field</param>
     /// <param name="TagCompanyName">Name of the company to check existance of tag</param>
-    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30])
+    /// <param name="SkipUpgrade">Sets the Skipped Upgrade field</param>
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30]; SkipUpgrade: Boolean): Boolean
```

Comment:
Could we remove this Boolean, it looks confusing? 
It would be better to have a dedicated function to check if the tag was skipped, it would be easier to use the API.

Reply:
I have removed the boolean and added the function HasUpgradeTagSkipped
But not sure if SetSkippedUpgrade in Upgrade Tag Impl. must also return boolean then.